packages/react-router/tests/router.test.tsx

```diff
@@ -374,7 +374,7 @@ describe('encoding: URL param segment for /posts/$slug', () => {
 
     await act(() => router.load())
 
-    expect(router.state.location.pathname).toBe('/posts/%F0%9F%9A%80')
+    expect(router.state.location.pathname).toBe('/posts/🚀')
```

Comment:
Not sure what needs to be changed here.

- Expectation of the test.
- Or the test name itself (i.e `it('should ...')`).